AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomGPU_BasicLevelSetup.py

```diff
@@ -0,0 +1,288 @@
+        secondary_grid_spacing_property = "Controller|Configuration|Secondary Grid Spacing"
+        secondary_grid_spacing_value = 1.0
+        grid_component.set_component_property_value(secondary_grid_spacing_property, secondary_grid_spacing_value)
+        secondary_grid_spacing_set = grid_component.get_component_property_value(
+            secondary_grid_spacing_property) == secondary_grid_spacing_value
+        Report.result(Tests.secondary_grid_spacing, secondary_grid_spacing_set)
+
+        # 5. Create Global Skylight (IBL) Entity as a child entity of the Default Level Entity.
+        global_skylight_name = "Global Skylight (IBL)"
+        global_skylight_entity = EditorEntity.create_editor_entity(global_skylight_name, default_level_entity.id)
+
+        # 6. Add HDRi Skybox component to the Global Skylight (IBL) Entity.
+        hdri_skybox_name = "HDRi Skybox"
+        hdri_skybox_component = global_skylight_entity.add_component(hdri_skybox_name)
+        Report.result(Tests.hdri_skybox_component_added, global_skylight_entity.has_component(hdri_skybox_name))
+
+        # 7. Add Global Skylight (IBL) component to the Global Skylight (IBL) Entity.
+        global_skylight_component = global_skylight_entity.add_component(global_skylight_name)
+        Report.result(Tests.global_skylight_component_added, global_skylight_entity.has_component(global_skylight_name))
+
+        # 8. Set the Cubemap Texture property of the HDRi Skybox component.
+        global_skylight_image_asset_path = os.path.join(
+            "LightingPresets", "greenwich_park_02_4k_iblskyboxcm_iblspecular.exr.streamingimage")
+        global_skylight_image_asset = Asset.find_asset_by_path(global_skylight_image_asset_path)
+        hdri_skybox_cubemap_texture_property = "Controller|Configuration|Cubemap Texture"
+        hdri_skybox_component.set_component_property_value(
+            hdri_skybox_cubemap_texture_property, global_skylight_image_asset)
+        Report.result(
+            Tests.hdri_skybox_cubemap_texture_set,
+            hdri_skybox_component.get_component_property_value(
+                hdri_skybox_cubemap_texture_property) == global_skylight_image_asset)
+
+        # 9. Set the Diffuse Image property of the Global Skylight (IBL) component.
+        # Re-use the same image that was used in the previous test step.
+        global_skylight_diffuse_image_property = "Controller|Configuration|Diffuse Image"
+        global_skylight_component.set_component_property_value(
+            global_skylight_diffuse_image_property, global_skylight_image_asset)
+        Report.result(
+            Tests.global_skylight_diffuse_image_set,
+            global_skylight_component.get_component_property_value(
+                global_skylight_diffuse_image_property) == global_skylight_image_asset)
+
+        # 10. Set the Specular Image property of the Global Skylight (IBL) component.
+        # Re-use the same image that was used in the previous test step.
```

Comment:
In the interest of using tests as a form of documentation, I suggest using the _specular.exr.streamingimage here. It comes from the same source .exr file

Reply:
ticket cut to deal with this, out of scope after investigating